sim/core/apl_values_aura.go

```diff
@@ -25,7 +22,7 @@ func (value *APLValueAuraIsActive) Type() proto.APLValueType {
 	return proto.APLValueType_ValueTypeBool
 }
 func (value *APLValueAuraIsActive) GetBool(sim *Simulation) bool {
-	return value.aura.Get().IsActive()
+	return value.aura.Get() != nil && value.aura.Get().IsActive()
```

Comment:
This fixes an issue where if you have an APL action with a condition on an aura that isn't available (e.g. rune not equipped, talent not taken, etc) the condition was evaluating to true and the action would be performed. In reality, the condition should evaluate to false and the action not taken


Example APL action for this
<img width="958" alt="image" src="https://github.com/wowsims/sod/assets/12898988/a748ff0c-63c4-47c7-8237-42804312e015">

Reply:
I dont agree with this change. Every apl value we have is treated as not existing when there is a warning in it (just like here youre trying to use a nonexistent aura). This would mean that this single value becomes special and does not behave like every other value. I dont like this inconsistency. Instead we can add a new apl value for “Has Rune” if we want to make handling missing runes better in apl.